presidio-image-anonymizer/tests/integration/test_image_anonymizer_engine.py

```diff
@@ -0,0 +1,26 @@
+from methods import get_resource_image, compare_images
+from presidio_image_anonymizer import ImageAnonymizerEngine
+
+
+def test_given_image_with_text_and_fill_then_text_is_greyed_out():
+    # Image with PII entities
+    image = get_resource_image("ocr_test.png")
+    result_image = get_resource_image("ocr_test_anonymized.png")
+    anonymized_image = ImageAnonymizerEngine().anonymize(image, 1)
+    assert compare_images(anonymized_image, result_image)
+
+
+def test_given_image_with_text_and_matrix_fill_then_text_is_colored_out():
+    # Image with PII entities
```

Comment:
missing some more info - what color did u select here (255, 160, 122) or 1 - maybe repalce with "cosnt string" representing th color, would make the test a bit more readable

Reply:
The expected result image has the color in it :) so anyone can open it up and look since it's in the project, but I can add it to the numbers as well.

Reply:
it will make the code easier o understand and will not require the user to go back to the image. As the unit test will act as code sample

Reply:
Done.